src/interfaces.ts

```diff
@@ -166,5 +168,4 @@ export interface IInputProps {
 
 export interface IEndpointOption {
   id: string;
-  name: string;
```

Comment:
Why fully remove name here? It looks like it's used in `getNameOrDefault`

Reply:
It is but since that can be overridden it MIGHT NOT HAVE NAME :o It could have... first_name? or something?